src/Microsoft.OpenApi/Any/OpenApiAny.cs

```diff
@@ -12,21 +12,21 @@ namespace Microsoft.OpenApi.Any
     /// </summary>
     public class OpenApiAny : IOpenApiElement, IOpenApiExtension
     {
-        private readonly JsonNode jsonNode;
+        private readonly JsonNode? jsonNode;
 
         /// <summary>
         /// Initializes the <see cref="OpenApiAny"/> class.
         /// </summary>
         /// <param name="jsonNode"></param>
-        public OpenApiAny(JsonNode jsonNode)
+        public OpenApiAny(JsonNode? jsonNode)
```

Comment:
why are we accepting a null node at all here? this is going to propagate all over the place.